Framework/Core/test/DummyMetricsService.h

```diff
@@ -1,28 +0,0 @@
-// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
Can we have a "no-op" implementation also for the `Monitoring` interface? It comes handy for tests.

Reply:
You mean a backend that does nothing? If so, of couse.

Reply:
yes, that's what I meant.

Reply:
`MonitoringFactory::Get("no-op://")` is available from v1.7.0